code/modules/paperwork/paper.dm

```diff
@@ -222,28 +222,60 @@
 /obj/item/paper/attack_ai(var/mob/living/silicon/ai/user)
 	show_content(user)
 
-/obj/item/paper/attack(mob/living/carbon/M as mob, mob/living/carbon/user as mob)
-	if(user.zone_sel.selecting == BP_EYES)
-		user.visible_message("<span class='notice'>You show the paper to [M]. </span>", \
-			"<span class='notice'> [user] holds up a paper and shows it to [M]. </span>")
+/obj/item/paper/attack(mob/living/carbon/M, mob/living/carbon/user)
+	if (user.zone_sel.selecting == BP_EYES)
+		user.visible_message(
+			SPAN_NOTICE("You show \the [src] to \the [M]. "), \
+			SPAN_NOTICE("\The [user] holds up \the [src] and shows it to \the [M].")
+		)
 		M.examinate(src)
 
-	else if(user.zone_sel.selecting == BP_MOUTH) // lipstick wiping
-		if(ishuman(M))
-			var/mob/living/carbon/human/H = M
-			if(H == user)
-				to_chat(user, "<span class='notice'>You wipe off the lipstick with [src].</span>")
+	else if (ishuman(M)) // check human-specific interactions
+		var/mob/living/carbon/human/H = M
+
+		if (user.zone_sel.selecting == BP_MOUTH) // lipstick wiping
+			if (H == user)
+				to_chat(user, SPAN_NOTICE("You wipe off the lipstick with \the [src]."))
```

Comment:
Same story as lines 260/261, since you're touching the code, why not make it a visible message?

Reply:
No particular preference one way or another - just because I missed it. I'll put 'er in.